test/test_functionalization.py

```diff
@@ -143,24 +144,20 @@ def forward(self, a_1):
     clone = torch.ops.aten.clone.default(view_copy);  view_copy = None
     view_copy_1 = torch.ops.aten.view_copy.default(clone, [16, 64, 128, 128]);  clone = None
     relu = torch.ops.aten.relu.default(view_copy_1);  view_copy_1 = None
-    sum_1 = torch.ops.aten.sum.default(relu)
+    view_copy_2 = torch.ops.aten.view_copy.default(relu, [1, 1024, 128, 128]);  relu = None
+    view_copy_3 = torch.ops.aten.view_copy.default(view_copy_2, [16, 64, 128, 128]);  view_copy_2 = None
+    sum_1 = torch.ops.aten.sum.default(view_copy_3)
```

Comment:
I'm still auditing the expect test changes.

One annoying source of changes is extra views that DCE won't eliminate automatically, but are technically unnecessary. In this example, `view_copy_3` is completely equivalent to `relu`.

I suppose we could get functionalization to figure out when this happens and "fix" it - do you think it would be reasonable for downstream consumers to optimize these away?

Reply:
The other major source of test wobbling is definitely DCE. Some deleted lines, which cause some variable names to wobble in the graph.

Reply:
Views should be optimized away, I think this is fine.